src/ripple/protocol/SecretKey.h

```diff
@@ -35,8 +35,11 @@ namespace ripple {
 class SecretKey
 {
 private:
-    std::uint8_t buf_[32];
+    std::array<std::uint8_t, 32> buf_;
 
+    friend
+    boost::optional<SecretKey>
+    parseBase58SecretKey (TokenType type, std::string const& s);
 public:
     using const_iterator = std::uint8_t const*;
```

Comment:
Should probably be `std::array<...>::const_iterator`

Reply:
Will do.

Reply:
Addressed in e71cfc80b1583e5055c9f4ed5828b2e7c7deeeb4